src/test/java/ContinuousIntegrationServerTest.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ContinuousIntegrationServerTest {
+
+    @Test
+    void cloneTheProject() {
```

Comment:
We said last last that for Lab 2 we will make test method with at most one assert per function. There you can make a function `cloneTheProject_HelloWorldExists` and `cloneTheProject_GoodbyeWorldDontExists`

Reply:
Yes, i will fix that!